src/osbuild-manifests/aws_tag.py

```diff
@@ -0,0 +1,59 @@
+import os
+import subprocess
+import json
+import argparse
+
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--stream', dest='stream', type=str, help='Fedora stream', required=True)
+    parser.add_argument('--arch', dest='arch', type=str, help='Architecture', default='x86_64')
+    args = parser.parse_args()
+
+    builds = getBuildsForStream(args.stream, args.arch)
+    for build in builds:
+        print("The build is "+build)

```

Comment:
to me it's easier to use f-strings throughout:

```suggestion
        print(f"The build is {build}")

```